packages/chain-adapters/src/ethereum/EthereumChainAdapter.ts

```diff
@@ -253,26 +253,51 @@ export class ChainAdapter implements IChainAdapter<ChainTypes.Ethereum> {
       data
     })
 
+    const feeData = (await this.providers.http.getGasFees()).data
+    const normalizationConstants = {
+      instant: String(new BigNumber(fees.instant).dividedBy(fees.fast)),
+      average: String(1),
+      slow: String(new BigNumber(fees.low).dividedBy(fees.fast))
+    }
+
     return {
       fast: {
         txFee: new BigNumber(fees.instant).times(gasLimit).toPrecision(),
         chainSpecific: {
           gasLimit,
-          gasPrice: String(fees.instant)
+          gasPrice: String(fees.instant),
+          maxFeePerGas: String(
+            new BigNumber(feeData.maxFeePerGas).times(normalizationConstants.instant)
+          ),
+          maxPriorityFeePerGas: String(
+            new BigNumber(feeData.maxPriorityFeePerGas).times(normalizationConstants.instant)
+          )
         }
       },
       average: {
         txFee: new BigNumber(fees.fast).times(gasLimit).toPrecision(),
```

Comment:
If eip fees confirmed working, it seems like we would want that to be our default for `txFee`